packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -33,14 +43,14 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     super(props);
     this._subscribers = new Set<Function>();
     this._stickies = new Set<Sticky>();
-
+    const { scrollbarVisibility } = this.props;
     this.state = {
       stickyTopHeight: 0,
       stickyBottomHeight: 0,
-      scrollbarWidth: 0,
-      scrollbarHeight: 0
+      scrollbarWidth: scrollbarVisibility === ScrollbarVisibility.always ? this._getScrollbarHeightFromLocalStorage(true) : 0,
```

Comment:
I thought deriving state from props was an anti-pattern we were looking to avoid.

https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#when-to-use-derived-state

Reply:
The state is being initialized here conditionally and props are not duplicated into state. I don't think there would a real life scenario which would change scrollbarVisibility prop but it is possible for the prop to get changed...